BuildToolsVersion.txt

```diff
@@ -1 +1 @@
-2.0.0-prerelease-01609-03
\ No newline at end of file
+2.0.0-prerelease-01721-01
```

Comment:
What do you think about moving this to `2.0.0-servicing-02026-05` - which is the latest buildtools from the release/2.0.0 branch?  It can be done separately, but I was thinking as long as we're in here, getting it in sync with the other repos would be a good idea.

Reply:
I do want to do that but I want to do it at the same time we move the CLI to 2.0.0 once we get that all working in BuildTools.

Reply:
I don't think it is necessary to wait until we move the CLI to 2.0.0.  corefx and coreclr release/2.0.0 is already on the `2.0.0-servicing-` versions of buildtools.

See https://github.com/dotnet/corefx/blob/dev/release/2.0.0/BuildToolsVersion.txt
https://github.com/dotnet/coreclr/blob/release/2.0.0/BuildToolsVersion.txt

And they aren't on CLI 2.0.0.

I **really** think we need to strive harder to make our toolsets consistent across repos.  What we are doing today isn't working very well IMO.

Reply:
OK you convinced me :) I will update this to match corefx/coreclr release/2.0.0 branches